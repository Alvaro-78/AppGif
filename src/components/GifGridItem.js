import React from 'react';

const GifGridItem = ({ title, url }) => {
  return (
    <>
      <div className='card animate__animated animate__fadeIn'>
        <img className='img' src={url} alt={title}></img>
        <p className='card-title'>{title}</p>
      </div>
    </>
  );
};

export default GifGridItem;
