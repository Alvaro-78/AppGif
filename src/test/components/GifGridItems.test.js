import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Testing component', () => {
  const title = 'algo';
  const url = 'http://localhost:3000/';

  it('should render component correctly', () => {
    const wrapper = shallow(<GifGridItem title={title} url={url} />);
    expect(wrapper).toMatchSnapshot();
  });
});
