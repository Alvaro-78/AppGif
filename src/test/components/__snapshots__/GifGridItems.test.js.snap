// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component should render component correctly 1`] = `
<Fragment>
  <div
    className="card animate__animated animate__fadeIn"
  >
    <img
      alt="algo"
      className="img"
      src="http://localhost:3000/"
    />
    <p
      className="card-title"
    >
      algo
    </p>
  </div>
</Fragment>
`;
